tests/Sandpit/Exscientia/Process/test_openmm/out_test_equilibrate_all_.py

```diff
@@ -0,0 +1,80 @@
+from openmm import *
```

Comment:
I don't think we need to include all the script in.

Just do one integration test where you test if openMM would actually run.

Then you add some small unit test that check if the key string is in the output.

Reply:
> Isn't it more rigorous to have the complete files checked?

I would imagine that the reason for checking is to ensure that openMM would run it fine. Which shall be covered by the integration test where you check if the atom moves.
The major downside of have the complete files checked is that say if we move from loading openMM system by amber file to some other format, all these scripts will needs to be regenerated.